pkg/internal/template_test.go

```diff
@@ -0,0 +1,251 @@
+package internal_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/buildpacks/scafall/pkg/internal"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/Netflix/go-expect"
+	h "github.com/buildpacks/pack/testhelpers"
+	pseudotty "github.com/creack/pty"
+	"github.com/hinshun/vt10x"
+	"github.com/sclevine/spec"
+)
+
+func testReadPrompt(t *testing.T, when spec.G, it spec.S) {
+	when("Reading a prompt file", func() {
+		it("reads a correct prompt file", func() {
+			tmpDir, _ := ioutil.TempDir("", "test")
```

Comment:
nit: You can use `t.TempDir()` to simplify this, and not require the explicit `os.RemoveAll`